tools/Vagrantfile

```diff
@@ -1,19 +1,30 @@
 # -*- mode: ruby -*-
```

Comment:
`tools` in composite tree? or should it be part of `cFE` tree in extern?

Reply:
It works for non cFE composite development. Open to changing this though.